### ISSUE TITLE
Does this formbuilder have maxtrix ?

### ISSUE THREAD

**Issue by Thisissparta100** [kevinchappell/formBuilder#1133]
Does this formbuilder have maxtrix inside? Or do you need to build it self with Control plugin ?

**Comment by Wujek280** [comment]
For a 2D matrix as a workarount you could set up textarea with subtype tinymce and insert table

**Comment by Thisissparta100** [comment]
@Wujek280 Thx for reaction no i want a metrix with radio-button

**Comment by Wujek280** [comment]
@Thisissparta100  I'm currently buildling custom control for matrix support in formBuilder and I have encountered problem with multiple inputs in same row - **value of field is always set to last modified input element** despite having hidden input value updated. Couldn't find workaround for this problem so far.

**Comment by Thisissparta100** [comment]
Can you share your code? Do you make it inside the plugin?

**Comment by stg-andrei** [comment]
> @Thisissparta100 I'm currently buildling custom control for matrix support in formBuilder and I have encountered problem with multiple inputs in same row - **value of field is always set to last modified input element** despite having hidden input value updated. Couldn't find workaround for this problem so far.

Any updates on this? still stuck?

**Comment by Wujek280** [comment]
@stg-andrei  I've resolved this problem in https://github.com/kevinchappell/formBuilder/issues/1148#issuecomment-744577228

**Comment by stg-andrei** [comment]
> @stg-andrei I've resolved this problem in [#1148 (comment)](https://github.com/kevinchappell/formBuilder/issues/1148#issuecomment-744577228)

Hey @Wujek280 , I assumed we were talking about a matrix table with radioboxes? Am I missing something? :)

**Comment by Wujek280** [comment]
Yes, but You've quoted my question about particular problem I've encounter during implementation of matrix custom element.

Before any questions arise, sorry to say but the solution I got is propietary to the company I am working for and I'm not allowed to share it 1:1

It might be not very useful for general purpose forms as it saves state to JSON stringified value to render itself

**Comment by stg-andrei** [comment]
> Yes, but You've quoted my question about particular problem I've encounter during implementation of matrix custom element.
> 
> Before any questions arise, sorry to say but the solution I got is propietary to the company I am working for and I'm not allowed to share it 1:1
> 
> It might be not very useful for general purpose forms as it saves state to JSON stringified value to render itself

Okay, no worries :)

**Comment by lucasnetau** [comment]
See https://github.com/lucasnetau/formBuilder-plugins/blob/main/plugins/grid.js
